src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -417,10 +421,14 @@ async def _start_running(self) -> None:
         ]
         # now we wait for the server to actually start
         await self._server_started.wait()
-        # let's run
+
+        # let's run with message queues set
```

Comment:
that comment should be redundant

Reply:
right :smile: that was more emotional comment than helpful really